test/runtests.jl

```diff
@@ -8,7 +8,7 @@ using RCall
 reval("install.packages('vegan')")
 
 @testset "Biobakery Utilities" begin
-    abund = import_abundance_table("metaphlan_test.tsv")
+    abund = import_abundance_table("dev/BiobakeryUtils/test/metaphlan_test.tsv")
```

Comment:
Tests are typically run with `test/` as the working directory, so this should stay as it is. As you've changed it here, the tests will fail when run in an automated way